python/py-sip4/Portfile

```diff
@@ -129,6 +129,7 @@ if {${name} ne ${subport}} {
 
     destroot.cmd        make
     destroot.destdir    DESTDIR=${destroot}
+    destroot.target     install
```

Comment:
As I said in the ticket, the problem is happening because the python portgroup now defaults to `python.pep517 yes` and it only sets `destroot.target install` when `python.pep517` is `no`. I'm not sure what else `python.pep517 yes` does, but it seems clear that this port does not want it. Maybe the solution should instead be for this port to set `python.pep517 no` explicitly?

Reply:
@ryandesign Let me try.

Reply:
UPD. Seems to work fine, rebased.